openff/toolkit/tests/test_molecule.py

```diff
@@ -914,7 +913,11 @@ def test_create_from_file(self):
         # Ensure that attempting to initialize a single Molecule from a file
         # containing multiple molecules raises a ValueError
         filename = get_data_file_path("molecules/butane_multi.sdf")
-        with pytest.raises(ValueError):
+
+        with pytest.raises(
+            ValueError,
+            match="Specified file or file-like.*exactly one molecule",
+        ):
```

Comment:
This is the traceback from triggering this test manually, which I find pretty confusing:

```
ValueError: Cannot construct openff.toolkit.topology.Molecule from /Users/mattthompson/software/openff-toolkit/openff/toolkit/data/molecules/butane_multi.sdf
No registered toolkits can provide the capability "from_object" for args "('/Users/mattthompson/software/openff-toolkit/openff/toolkit/data/molecules/butane_multi.sdf',)" and kwargs "{'allow_undefined_stereo': True, '_cls': <class 'openff.toolkit.topology.molecule.Molecule'>}"
Available toolkits are: [ToolkitWrapper around OpenEye Toolkit version 2022.1.1, ToolkitWrapper around The RDKit version 2022.09.1, ToolkitWrapper around AmberTools version 22.0, ToolkitWrapper around Built-in Toolkit version None]
 ToolkitWrapper around OpenEye Toolkit version 2022.1.1 <class 'NotImplementedError'> : Cannot create Molecule from <class 'str'> object
 ToolkitWrapper around The RDKit version 2022.09.1 <class 'NotImplementedError'> : Cannot create Molecule from <class 'str'> object
Specified file or file-like object must contain exactly one molecule
```

It's raised from [here](https://github.com/openforcefield/openff-toolkit/blob/ff6b37166ecf7c01f5b75978adf1c24a32d2e632/openff/toolkit/topology/molecule.py#L982-L985) but the error message adds extra information about toolkit wrappers not being able to create molecules from strings, which happens earlier in guessing whether or not the input (a `str` representing a file path) is within the types that can be parsed.